src/app/[lang]/consultation/gecitkale/_components/SqueezeConsultationForm.tsx

```
@@ -99,17 +78,17 @@ export default function SqueezeConsultationForm({ translations, nonce }: Squeeze
-      email: Yup.string().email('Invalid email address').required('Email is required'),
-      phoneNumber: Yup.string().required('Phone number is required'),
-      interestReason: Yup.string().required('Please select an interest reason'),
+      firstName: Yup.string().required(content.form.validationMessages.firstName),
+      lastName: Yup.string().required(content.form.validationMessages.lastName),
+      email: Yup.string().email(content.form.validationMessages.emailInvalid).required(content.form.validationMessages.email),
+      phoneNumber: Yup.string().required(content.form.validationMessages.phoneNumber),
+      interestReason: Yup.string().required(content.form.validationMessages.interestReason),
       interestReasonOther: Yup.string().when('interestReason', {
         is: 'other',
-        then: (schema) => schema.required('Please specify your reason'),
+        then: (schema) => schema.required(content.form.validationMessages.interestReasonOther),
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)